lib/sprockets/sassc_processor.rb

```diff
@@ -42,7 +42,10 @@ def initialize(options = {}, &block)
       @cache_key = "#{self.class.name}:#{VERSION}:#{Autoload::SassC::VERSION}:#{@cache_version}".freeze
       @importer_class = options[:importer]
       @sass_config = options[:sass_config] || {}
+      module_include_functions = Gem::Version.new(Autoload::SassC::VERSION) < Gem::Version.new('2.3.0')
+      @module_include_functions = module_include_functions
       @functions = Module.new do
+        include Autoload::SassC::Script::Functions unless module_include_functions
```

Comment:
I just realized there is a previous PR on this: https://github.com/rails/sprockets/pull/630

This PR is more or less the same, but this line is a detail that the previous PR missed.